azurerm_network_manager_deployment: polling fails with empty Content-Type on provider 4.74.0+

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Community Note

<!--- Please keep this note for the community --->

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave comments along the lines of "+1", "me too" or "any updates", they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment and review the [contribution guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/README.md) to help.

<!--- Thank you for keeping this note for the community --->


### Terraform Version

 1.14.3

### AzureRM Provider Version

4.76.0

### Affected Resource(s)/Data Source(s)

azurerm_network_manager_deployment

### Terraform Configuration Files

```hcl
resource "azurerm_network_manager" "avnm" {
  resource_group_name = "rg-resources"
  name                = "avnm-chn-hccase"
  location            = "switzerlandnorth"
  description         = "test"
  scope_accesses      = ["Connectivity"]
  scope {
    subscription_ids = [data.azurerm_subscription.current.id]
  }
}

resource "azurerm_network_manager_network_group" "avnm_network_groups" {
  name               = "test"
  description        = "test"
  network_manager_id = azurerm_network_manager.avnm.id
}

resource "azurerm_network_manager_connectivity_configuration" "avnm_connectivity" {
  name                  = "test"
  connectivity_topology = "HubAndSpoke"
  network_manager_id    = azurerm_network_manager.avnm.id
  applies_to_group {
    group_connectivity = "None"
    network_group_id   = azurerm_network_manager_network_group.avnm_network_groups.id
    use_hub_gateway    = true
  }
  hub {
    resource_id   = module.vnet.virtual_network_id
    resource_type = "Microsoft.Network/virtualNetworks"
  }
  delete_existing_peering_enabled = false
}

resource "azurerm_network_manager_deployment" "connectivity" {
  scope_access       = "Connectivity"
  configuration_ids  = [azurerm_network_manager_connectivity_configuration.avnm_connectivity.id]
  location           = "switzerlandnorth"
  network_manager_id = azurerm_network_manager.avnm.id
}
```

### Debug Output/Panic Output

```shell
2026-06-05T12:22:39.499+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: [DEBUG] AzureRM Request: 
2026-06-05T12:22:39.499+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: POST /subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/commit?api-version=2025-01-01 HTTP/1.1
2026-06-05T12:22:39.499+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Host: management.azure.com
2026-06-05T12:22:39.499+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: User-Agent: HashiCorp/go-azure-sdk (Go-http-Client/1.1 networkmanagers/2025-01-01) HashiCorp Terraform/1.14.3 (+https://www.terraform.io) terraform-provider-azurerm/4.76.0 pid-222c6c49-1b0a-5959-a213-6608f9eb8820
2026-06-05T12:22:39.499+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Content-Length: 263
2026-06-05T12:22:39.499+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Content-Type: application/json; charset=utf-8
2026-06-05T12:22:39.499+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Correlation-Request-Id: 3e2f3edd-6683-9da0-6d7e-25492b5ec43c
2026-06-05T12:22:39.499+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Accept-Encoding: gzip
2026-06-05T12:22:39.499+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5
2026-06-05T12:22:39.499+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: {"commitType":"Connectivity","configurationIds":["/subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/connectivityConfigurations/test"],"targetLocations":["switzerlandnorth"]}
2026-06-05T12:22:39.499+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: [DEBUG] POST https://management.azure.com/subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/commit?api-version=2025-01-01
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: [DEBUG] AzureRM Response for https://management.azure.com/subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/commit?api-version=2025-01-01: 
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: HTTP/2.0 202 Accepted
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Content-Length: 460
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Cache-Control: no-cache
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Content-Type: application/json; charset=utf-8
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Date: Fri, 05 Jun 2026 10:22:40 GMT
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Expires: -1
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Location: https://management.azure.com/subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/operationResults/00000000-0000-0000-0000-000000000000?api-version=2025-01-01&t=639162517611290374&c=MIIHlTCCBn2gAwIBAgIRAKD76JWTnyyPjJFM0EstdxQwDQYJKoZIhvcNAQELBQAwNjE0MDIGA1UEAxMrQ0NNRSBHMSBUTFMgUlNBIDIwNDggU0hBMjU2IDIwNDkgRVVTMiBDQSAwMTAeFw0yNjA0MTMxMDM4MjlaFw0yNjEwMDgxNjM4MjlaMEAxPjA8BgNVBAMTNWFzeW5jb3BlcmF0aW9uc2lnbmluZ2NlcnRpZmljYXRlLm1hbmFnZW1lbnQuYXp1cmUuY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAu8o-sTL0AvwO0e7AAJHaMVEzHg_owu2xGyIrXRZ3T6e6vsEFQL7YB7zo6M2z2a4QB3wB_8hM9o6tOE6WCS5Msg8sWypwZIFMnbP4RETaa-rOJp7PjinHUJbhtGTZ7mZAT4npIz0q2-WAfdL8KzPkwF1mWO6G5upMZGGEr5Cgl8U3qeVcvSzKAFe9p4K3nT03s-1mQ8I4ryQqX_4PCzk2tmlHREcslMzMUCHIZuTkZKxmWs_DPRGhfzP5YZxd7J6imzoPipcbkiMFPdLeHBojPaQ0IdaWkKqax1lRCkSAW8oVvGc95DbAYFcOIDFgODKxe40b-jQEV6oyOeExNP3PzQIDAQABo4IEkjCCBI4wgZ0GA1UdIASBlTCBkjAMBgorBgEEAYI3ewEBMGYGCisGAQQBgjd7AgIwWDBWBggrBgEFBQcCAjBKHkgAMwAzAGUAMAAxADkAMgAxAC0ANABkADYANAAtADQAZgA4AGMALQBhADAANQA1AC0ANQBiAGQAYQBmAGYAZAA1AGUAMwAzAGQwDAYKKwYBBAGCN3sDAjAMBgorBgEEAYI3ewQCMAwGA1UdEwEB_wQCMAAwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMA4GA1UdDwEB_wQEAwIFoDAdBgNVHQ4EFgQUnDSza4PUM-UYRhQrDE3szwJXw7MwHwYDVR0jBBgwFoAU_Ow-26p8H4IeBbihBvlD5wKzCrkwggGyBgNVHR8EggGpMIIBpTBpoGegZYZjaHR0cDovL3ByaW1hcnktY2RuLnBraS5jb3JlLndpbmRvd3MubmV0L2Vhc3R1czIvY3Jscy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxLzczL2N1cnJlbnQuY3JsMGugaaBnhmVodHRwOi8vc2Vjb25kYXJ5LWNkbi5wa2kuY29yZS53aW5kb3dzLm5ldC9lYXN0dXMyL2NybHMvY2NtZWVhc3R1czJwa2kvY2NtZWVhc3R1czJpY2EwMS83My9jdXJyZW50LmNybDBaoFigVoZUaHR0cDovL2NybC5taWNyb3NvZnQuY29tL2Vhc3R1czIvY3Jscy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxLzczL2N1cnJlbnQuY3JsMG-gbaBrhmlodHRwOi8vY2NtZWVhc3R1czJwa2kuZWFzdHVzMi5wa2kuY29yZS53aW5kb3dzLm5ldC9jZXJ0aWZpY2F0ZUF1dGhvcml0aWVzL2NjbWVlYXN0dXMyaWNhMDEvNzMvY3VycmVudC5jcmwwggG3BggrBgEFBQcBAQSCAakwggGlMGwGCCsGAQUFBzAChmBodHRwOi8vcHJpbWFyeS1jZG4ucGtpLmNvcmUud2luZG93cy5uZXQvZWFzdHVzMi9jYWNlcnRzL2NjbWVlYXN0dXMycGtpL2NjbWVlYXN0dXMyaWNhMDEvY2VydC5jZXIwbgYIKwYBBQUHMAKGYmh0dHA6Ly9zZWNvbmRhcnktY2RuLnBraS5jb3JlLndpbmRvd3MubmV0L2Vhc3R1czIvY2FjZXJ0cy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxL2NlcnQuY2VyMF0GCCsGAQUFBzAChlFodHRwOi8vY3JsLm1pY3Jvc29mdC5jb20vZWFzdHVzMi9jYWNlcnRzL2NjbWVlYXN0dXMycGtpL2NjbWVlYXN0dXMyaWNhMDEvY2VydC5jZXIwZgYIKwYBBQUHMAKGWmh0dHA6Ly9jY21lZWFzdHVzMnBraS5lYXN0dXMyLnBraS5jb3JlLndpbmRvd3MubmV0L2NlcnRpZmljYXRlQXV0aG9yaXRpZXMvY2NtZWVhc3R1czJpY2EwMTANBgkqhkiG9w0BAQsFAAOCAQEAMTXNm7f9ecpkYKw26JGy3QtSP16FypZTMbHp9xQm6p1LKaX1HMCiezhGWVV_qV9eesoavFBooGOWV3Cuj2blsw9UDCQ1M_D5vw0oHoSyTt-aeXoYupM8VNflVeIJGYw1GvaGFxTX1JkfH7xqufgkddmhrqAKqpimQLE34HIz0jMMfG0zd6rsyBKOFYljQXuhFA5Fwz-tQAU-XJbYFqavV-rrLq4QjqEAKVQHL3gwnfXmsJJFS_ucsdBF2GoeO6il9QrCzp8S2qVufWVvnFn4RQKz_VMkQaUbf_sl4QwXb0ZwvSV6tL2UV7B7ByFNWihiXwPmY7MJ8K094lJx6SR1Wg&s=<Redacted>=reySbYbADS6BB_GL3gBKA2pzyjzRwP7gsOdpvwMfMbQ
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Pragma: no-cache
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Strict-Transport-Security: max-age=31536000; includeSubDomains
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Cache: CONFIG_NOCACHE
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Content-Type-Options: nosniff
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Correlation-Request-Id: 3e2f3edd-6683-9da0-6d7e-25492b5ec43c
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Operation-Identifier: appId=<Redacted>,tenantId=<Redacted>,objectId=<Redacted>/switzerlandnorth/db32ffc3-44e7-4aa7-8d4f-a8f5bc03bfb5
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Ratelimit-Remaining-Subscription-Global-Writes: 2999
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Ratelimit-Remaining-Subscription-Writes: 199
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Request-Id: b25b0d16-bd78-4017-a0bb-d0d1b1b02bcf
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Routing-Request-Id: SWITZERLANDNORTH:20260605T102241Z:b25b0d16-bd78-4017-a0bb-d0d1b1b02bcf
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Msedge-Ref: Ref A: 41C8FC73DDB9428086D4E3E403AB72C3 Ref B: GVA201070812029 Ref C: 2026-06-05T10:22:39Z
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5
2026-06-05T12:22:41.149+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: {"configurationIds":["/subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/connectivityConfigurations/test"],"targetLocations":["switzerlandnorth"],"commitType":"Connectivity","commitId":"/subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/commit/82061c57-f427-4967-989a-b6dd7df3f7be"}
2026-06-05T12:22:43.145+0200 [TRACE] dag/walk: vertex "provider[\"registry.terraform.io/hashicorp/azurerm\"] (close)" is waiting for "azurerm_network_manager_deployment.connectivity"
2026-06-05T12:22:43.145+0200 [TRACE] dag/walk: vertex "root" is waiting for "provider[\"registry.terraform.io/hashicorp/azurerm\"] (close)"
2026-06-05T12:22:48.148+0200 [TRACE] dag/walk: vertex "root" is waiting for "provider[\"registry.terraform.io/hashicorp/azurerm\"] (close)"
2026-06-05T12:22:48.148+0200 [TRACE] dag/walk: vertex "provider[\"registry.terraform.io/hashicorp/azurerm\"] (close)" is waiting for "azurerm_network_manager_deployment.connectivity"
{"@level":"info","@message":"azurerm_network_manager_deployment.connectivity: Still creating... [10s elapsed]","@module":"terraform.ui","@timestamp":"2026-06-05T12:22:49.064016+02:00","hook":{"resource":{"addr":"azurerm_network_manager_deployment.connectivity","module":"","resource":"azurerm_network_manager_deployment.connectivity","implied_provider":"azurerm","resource_type":"azurerm_network_manager_deployment","resource_name":"connectivity","resource_key":null},"action":"create","elapsed_seconds":10},"type":"apply_progress"}
2026-06-05T12:22:51.156+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: [DEBUG] AzureRM Request: 
2026-06-05T12:22:51.156+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: GET /subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/operationResults/00000000-0000-0000-0000-000000000000?api-version=2025-01-01&t=639162517611290374&c=MIIHlTCCBn2gAwIBAgIRAKD76JWTnyyPjJFM0EstdxQwDQYJKoZIhvcNAQELBQAwNjE0MDIGA1UEAxMrQ0NNRSBHMSBUTFMgUlNBIDIwNDggU0hBMjU2IDIwNDkgRVVTMiBDQSAwMTAeFw0yNjA0MTMxMDM4MjlaFw0yNjEwMDgxNjM4MjlaMEAxPjA8BgNVBAMTNWFzeW5jb3BlcmF0aW9uc2lnbmluZ2NlcnRpZmljYXRlLm1hbmFnZW1lbnQuYXp1cmUuY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAu8o-sTL0AvwO0e7AAJHaMVEzHg_owu2xGyIrXRZ3T6e6vsEFQL7YB7zo6M2z2a4QB3wB_8hM9o6tOE6WCS5Msg8sWypwZIFMnbP4RETaa-rOJp7PjinHUJbhtGTZ7mZAT4npIz0q2-WAfdL8KzPkwF1mWO6G5upMZGGEr5Cgl8U3qeVcvSzKAFe9p4K3nT03s-1mQ8I4ryQqX_4PCzk2tmlHREcslMzMUCHIZuTkZKxmWs_DPRGhfzP5YZxd7J6imzoPipcbkiMFPdLeHBojPaQ0IdaWkKqax1lRCkSAW8oVvGc95DbAYFcOIDFgODKxe40b-jQEV6oyOeExNP3PzQIDAQABo4IEkjCCBI4wgZ0GA1UdIASBlTCBkjAMBgorBgEEAYI3ewEBMGYGCisGAQQBgjd7AgIwWDBWBggrBgEFBQcCAjBKHkgAMwAzAGUAMAAxADkAMgAxAC0ANABkADYANAAtADQAZgA4AGMALQBhADAANQA1AC0ANQBiAGQAYQBmAGYAZAA1AGUAMwAzAGQwDAYKKwYBBAGCN3sDAjAMBgorBgEEAYI3ewQCMAwGA1UdEwEB_wQCMAAwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMA4GA1UdDwEB_wQEAwIFoDAdBgNVHQ4EFgQUnDSza4PUM-UYRhQrDE3szwJXw7MwHwYDVR0jBBgwFoAU_Ow-26p8H4IeBbihBvlD5wKzCrkwggGyBgNVHR8EggGpMIIBpTBpoGegZYZjaHR0cDovL3ByaW1hcnktY2RuLnBraS5jb3JlLndpbmRvd3MubmV0L2Vhc3R1czIvY3Jscy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxLzczL2N1cnJlbnQuY3JsMGugaaBnhmVodHRwOi8vc2Vjb25kYXJ5LWNkbi5wa2kuY29yZS53aW5kb3dzLm5ldC9lYXN0dXMyL2NybHMvY2NtZWVhc3R1czJwa2kvY2NtZWVhc3R1czJpY2EwMS83My9jdXJyZW50LmNybDBaoFigVoZUaHR0cDovL2NybC5taWNyb3NvZnQuY29tL2Vhc3R1czIvY3Jscy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxLzczL2N1cnJlbnQuY3JsMG-gbaBrhmlodHRwOi8vY2NtZWVhc3R1czJwa2kuZWFzdHVzMi5wa2kuY29yZS53aW5kb3dzLm5ldC9jZXJ0aWZpY2F0ZUF1dGhvcml0aWVzL2NjbWVlYXN0dXMyaWNhMDEvNzMvY3VycmVudC5jcmwwggG3BggrBgEFBQcBAQSCAakwggGlMGwGCCsGAQUFBzAChmBodHRwOi8vcHJpbWFyeS1jZG4ucGtpLmNvcmUud2luZG93cy5uZXQvZWFzdHVzMi9jYWNlcnRzL2NjbWVlYXN0dXMycGtpL2NjbWVlYXN0dXMyaWNhMDEvY2VydC5jZXIwbgYIKwYBBQUHMAKGYmh0dHA6Ly9zZWNvbmRhcnktY2RuLnBraS5jb3JlLndpbmRvd3MubmV0L2Vhc3R1czIvY2FjZXJ0cy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxL2NlcnQuY2VyMF0GCCsGAQUFBzAChlFodHRwOi8vY3JsLm1pY3Jvc29mdC5jb20vZWFzdHVzMi9jYWNlcnRzL2NjbWVlYXN0dXMycGtpL2NjbWVlYXN0dXMyaWNhMDEvY2VydC5jZXIwZgYIKwYBBQUHMAKGWmh0dHA6Ly9jY21lZWFzdHVzMnBraS5lYXN0dXMyLnBraS5jb3JlLndpbmRvd3MubmV0L2NlcnRpZmljYXRlQXV0aG9yaXRpZXMvY2NtZWVhc3R1czJpY2EwMTANBgkqhkiG9w0BAQsFAAOCAQEAMTXNm7f9ecpkYKw26JGy3QtSP16FypZTMbHp9xQm6p1LKaX1HMCiezhGWVV_qV9eesoavFBooGOWV3Cuj2blsw9UDCQ1M_D5vw0oHoSyTt-aeXoYupM8VNflVeIJGYw1GvaGFxTX1JkfH7xqufgkddmhrqAKqpimQLE34HIz0jMMfG0zd6rsyBKOFYljQXuhFA5Fwz-tQAU-XJbYFqavV-rrLq4QjqEAKVQHL3gwnfXmsJJFS_ucsdBF2GoeO6il9QrCzp8S2qVufWVvnFn4RQKz_VMkQaUbf_sl4QwXb0ZwvSV6tL2UV7B7ByFNWihiXwPmY7MJ8K094lJx6SR1Wg&s=<Redacted>=reySbYbADS6BB_GL3gBKA2pzyjzRwP7gsOdpvwMfMbQ HTTP/1.1
2026-06-05T12:22:51.156+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Host: management.azure.com
2026-06-05T12:22:51.156+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: User-Agent: HashiCorp/go-azure-sdk (Go-http-Client/1.1 networkmanagers/2025-01-01) HashiCorp Terraform/1.14.3 (+https://www.terraform.io) terraform-provider-azurerm/4.76.0 pid-222c6c49-1b0a-5959-a213-6608f9eb8820
2026-06-05T12:22:51.156+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Content-Type: application/json; charset=utf-8
2026-06-05T12:22:51.156+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Correlation-Request-Id: 3e2f3edd-6683-9da0-6d7e-25492b5ec43c
2026-06-05T12:22:51.156+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Accept-Encoding: gzip
2026-06-05T12:22:51.156+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5
2026-06-05T12:22:51.156+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5
2026-06-05T12:22:51.156+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: [DEBUG] GET https://management.azure.com/subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/operationResults/00000000-0000-0000-0000-000000000000?api-version=2025-01-01&t=639162517611290374&c=MIIHlTCCBn2gAwIBAgIRAKD76JWTnyyPjJFM0EstdxQwDQYJKoZIhvcNAQELBQAwNjE0MDIGA1UEAxMrQ0NNRSBHMSBUTFMgUlNBIDIwNDggU0hBMjU2IDIwNDkgRVVTMiBDQSAwMTAeFw0yNjA0MTMxMDM4MjlaFw0yNjEwMDgxNjM4MjlaMEAxPjA8BgNVBAMTNWFzeW5jb3BlcmF0aW9uc2lnbmluZ2NlcnRpZmljYXRlLm1hbmFnZW1lbnQuYXp1cmUuY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAu8o-sTL0AvwO0e7AAJHaMVEzHg_owu2xGyIrXRZ3T6e6vsEFQL7YB7zo6M2z2a4QB3wB_8hM9o6tOE6WCS5Msg8sWypwZIFMnbP4RETaa-rOJp7PjinHUJbhtGTZ7mZAT4npIz0q2-WAfdL8KzPkwF1mWO6G5upMZGGEr5Cgl8U3qeVcvSzKAFe9p4K3nT03s-1mQ8I4ryQqX_4PCzk2tmlHREcslMzMUCHIZuTkZKxmWs_DPRGhfzP5YZxd7J6imzoPipcbkiMFPdLeHBojPaQ0IdaWkKqax1lRCkSAW8oVvGc95DbAYFcOIDFgODKxe40b-jQEV6oyOeExNP3PzQIDAQABo4IEkjCCBI4wgZ0GA1UdIASBlTCBkjAMBgorBgEEAYI3ewEBMGYGCisGAQQBgjd7AgIwWDBWBggrBgEFBQcCAjBKHkgAMwAzAGUAMAAxADkAMgAxAC0ANABkADYANAAtADQAZgA4AGMALQBhADAANQA1AC0ANQBiAGQAYQBmAGYAZAA1AGUAMwAzAGQwDAYKKwYBBAGCN3sDAjAMBgorBgEEAYI3ewQCMAwGA1UdEwEB_wQCMAAwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMA4GA1UdDwEB_wQEAwIFoDAdBgNVHQ4EFgQUnDSza4PUM-UYRhQrDE3szwJXw7MwHwYDVR0jBBgwFoAU_Ow-26p8H4IeBbihBvlD5wKzCrkwggGyBgNVHR8EggGpMIIBpTBpoGegZYZjaHR0cDovL3ByaW1hcnktY2RuLnBraS5jb3JlLndpbmRvd3MubmV0L2Vhc3R1czIvY3Jscy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxLzczL2N1cnJlbnQuY3JsMGugaaBnhmVodHRwOi8vc2Vjb25kYXJ5LWNkbi5wa2kuY29yZS53aW5kb3dzLm5ldC9lYXN0dXMyL2NybHMvY2NtZWVhc3R1czJwa2kvY2NtZWVhc3R1czJpY2EwMS83My9jdXJyZW50LmNybDBaoFigVoZUaHR0cDovL2NybC5taWNyb3NvZnQuY29tL2Vhc3R1czIvY3Jscy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxLzczL2N1cnJlbnQuY3JsMG-gbaBrhmlodHRwOi8vY2NtZWVhc3R1czJwa2kuZWFzdHVzMi5wa2kuY29yZS53aW5kb3dzLm5ldC9jZXJ0aWZpY2F0ZUF1dGhvcml0aWVzL2NjbWVlYXN0dXMyaWNhMDEvNzMvY3VycmVudC5jcmwwggG3BggrBgEFBQcBAQSCAakwggGlMGwGCCsGAQUFBzAChmBodHRwOi8vcHJpbWFyeS1jZG4ucGtpLmNvcmUud2luZG93cy5uZXQvZWFzdHVzMi9jYWNlcnRzL2NjbWVlYXN0dXMycGtpL2NjbWVlYXN0dXMyaWNhMDEvY2VydC5jZXIwbgYIKwYBBQUHMAKGYmh0dHA6Ly9zZWNvbmRhcnktY2RuLnBraS5jb3JlLndpbmRvd3MubmV0L2Vhc3R1czIvY2FjZXJ0cy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxL2NlcnQuY2VyMF0GCCsGAQUFBzAChlFodHRwOi8vY3JsLm1pY3Jvc29mdC5jb20vZWFzdHVzMi9jYWNlcnRzL2NjbWVlYXN0dXMycGtpL2NjbWVlYXN0dXMyaWNhMDEvY2VydC5jZXIwZgYIKwYBBQUHMAKGWmh0dHA6Ly9jY21lZWFzdHVzMnBraS5lYXN0dXMyLnBraS5jb3JlLndpbmRvd3MubmV0L2NlcnRpZmljYXRlQXV0aG9yaXRpZXMvY2NtZWVhc3R1czJpY2EwMTANBgkqhkiG9w0BAQsFAAOCAQEAMTXNm7f9ecpkYKw26JGy3QtSP16FypZTMbHp9xQm6p1LKaX1HMCiezhGWVV_qV9eesoavFBooGOWV3Cuj2blsw9UDCQ1M_D5vw0oHoSyTt-aeXoYupM8VNflVeIJGYw1GvaGFxTX1JkfH7xqufgkddmhrqAKqpimQLE34HIz0jMMfG0zd6rsyBKOFYljQXuhFA5Fwz-tQAU-XJbYFqavV-rrLq4QjqEAKVQHL3gwnfXmsJJFS_ucsdBF2GoeO6il9QrCzp8S2qVufWVvnFn4RQKz_VMkQaUbf_sl4QwXb0ZwvSV6tL2UV7B7ByFNWihiXwPmY7MJ8K094lJx6SR1Wg&s=<Redacted>=reySbYbADS6BB_GL3gBKA2pzyjzRwP7gsOdpvwMfMbQ
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: [DEBUG] AzureRM Response for https://management.azure.com/subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/operationResults/00000000-0000-0000-0000-000000000000?api-version=2025-01-01&t=639162517611290374&c=MIIHlTCCBn2gAwIBAgIRAKD76JWTnyyPjJFM0EstdxQwDQYJKoZIhvcNAQELBQAwNjE0MDIGA1UEAxMrQ0NNRSBHMSBUTFMgUlNBIDIwNDggU0hBMjU2IDIwNDkgRVVTMiBDQSAwMTAeFw0yNjA0MTMxMDM4MjlaFw0yNjEwMDgxNjM4MjlaMEAxPjA8BgNVBAMTNWFzeW5jb3BlcmF0aW9uc2lnbmluZ2NlcnRpZmljYXRlLm1hbmFnZW1lbnQuYXp1cmUuY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAu8o-sTL0AvwO0e7AAJHaMVEzHg_owu2xGyIrXRZ3T6e6vsEFQL7YB7zo6M2z2a4QB3wB_8hM9o6tOE6WCS5Msg8sWypwZIFMnbP4RETaa-rOJp7PjinHUJbhtGTZ7mZAT4npIz0q2-WAfdL8KzPkwF1mWO6G5upMZGGEr5Cgl8U3qeVcvSzKAFe9p4K3nT03s-1mQ8I4ryQqX_4PCzk2tmlHREcslMzMUCHIZuTkZKxmWs_DPRGhfzP5YZxd7J6imzoPipcbkiMFPdLeHBojPaQ0IdaWkKqax1lRCkSAW8oVvGc95DbAYFcOIDFgODKxe40b-jQEV6oyOeExNP3PzQIDAQABo4IEkjCCBI4wgZ0GA1UdIASBlTCBkjAMBgorBgEEAYI3ewEBMGYGCisGAQQBgjd7AgIwWDBWBggrBgEFBQcCAjBKHkgAMwAzAGUAMAAxADkAMgAxAC0ANABkADYANAAtADQAZgA4AGMALQBhADAANQA1AC0ANQBiAGQAYQBmAGYAZAA1AGUAMwAzAGQwDAYKKwYBBAGCN3sDAjAMBgorBgEEAYI3ewQCMAwGA1UdEwEB_wQCMAAwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMA4GA1UdDwEB_wQEAwIFoDAdBgNVHQ4EFgQUnDSza4PUM-UYRhQrDE3szwJXw7MwHwYDVR0jBBgwFoAU_Ow-26p8H4IeBbihBvlD5wKzCrkwggGyBgNVHR8EggGpMIIBpTBpoGegZYZjaHR0cDovL3ByaW1hcnktY2RuLnBraS5jb3JlLndpbmRvd3MubmV0L2Vhc3R1czIvY3Jscy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxLzczL2N1cnJlbnQuY3JsMGugaaBnhmVodHRwOi8vc2Vjb25kYXJ5LWNkbi5wa2kuY29yZS53aW5kb3dzLm5ldC9lYXN0dXMyL2NybHMvY2NtZWVhc3R1czJwa2kvY2NtZWVhc3R1czJpY2EwMS83My9jdXJyZW50LmNybDBaoFigVoZUaHR0cDovL2NybC5taWNyb3NvZnQuY29tL2Vhc3R1czIvY3Jscy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxLzczL2N1cnJlbnQuY3JsMG-gbaBrhmlodHRwOi8vY2NtZWVhc3R1czJwa2kuZWFzdHVzMi5wa2kuY29yZS53aW5kb3dzLm5ldC9jZXJ0aWZpY2F0ZUF1dGhvcml0aWVzL2NjbWVlYXN0dXMyaWNhMDEvNzMvY3VycmVudC5jcmwwggG3BggrBgEFBQcBAQSCAakwggGlMGwGCCsGAQUFBzAChmBodHRwOi8vcHJpbWFyeS1jZG4ucGtpLmNvcmUud2luZG93cy5uZXQvZWFzdHVzMi9jYWNlcnRzL2NjbWVlYXN0dXMycGtpL2NjbWVlYXN0dXMyaWNhMDEvY2VydC5jZXIwbgYIKwYBBQUHMAKGYmh0dHA6Ly9zZWNvbmRhcnktY2RuLnBraS5jb3JlLndpbmRvd3MubmV0L2Vhc3R1czIvY2FjZXJ0cy9jY21lZWFzdHVzMnBraS9jY21lZWFzdHVzMmljYTAxL2NlcnQuY2VyMF0GCCsGAQUFBzAChlFodHRwOi8vY3JsLm1pY3Jvc29mdC5jb20vZWFzdHVzMi9jYWNlcnRzL2NjbWVlYXN0dXMycGtpL2NjbWVlYXN0dXMyaWNhMDEvY2VydC5jZXIwZgYIKwYBBQUHMAKGWmh0dHA6Ly9jY21lZWFzdHVzMnBraS5lYXN0dXMyLnBraS5jb3JlLndpbmRvd3MubmV0L2NlcnRpZmljYXRlQXV0aG9yaXRpZXMvY2NtZWVhc3R1czJpY2EwMTANBgkqhkiG9w0BAQsFAAOCAQEAMTXNm7f9ecpkYKw26JGy3QtSP16FypZTMbHp9xQm6p1LKaX1HMCiezhGWVV_qV9eesoavFBooGOWV3Cuj2blsw9UDCQ1M_D5vw0oHoSyTt-aeXoYupM8VNflVeIJGYw1GvaGFxTX1JkfH7xqufgkddmhrqAKqpimQLE34HIz0jMMfG0zd6rsyBKOFYljQXuhFA5Fwz-tQAU-XJbYFqavV-rrLq4QjqEAKVQHL3gwnfXmsJJFS_ucsdBF2GoeO6il9QrCzp8S2qVufWVvnFn4RQKz_VMkQaUbf_sl4QwXb0ZwvSV6tL2UV7B7ByFNWihiXwPmY7MJ8K094lJx6SR1Wg&s=<Redacted>=reySbYbADS6BB_GL3gBKA2pzyjzRwP7gsOdpvwMfMbQ: 
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: HTTP/2.0 204 No Content
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Connection: close
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Cache-Control: no-cache
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Date: Fri, 05 Jun 2026 10:22:51 GMT
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Expires: -1
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Pragma: no-cache
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: Strict-Transport-Security: max-age=31536000; includeSubDomains
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Cache: CONFIG_NOCACHE
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Content-Type-Options: nosniff
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Correlation-Request-Id: 3e2f3edd-6683-9da0-6d7e-25492b5ec43c
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Operation-Identifier: appId=<Redacted>,tenantId=<Redacted>,objectId=<Redacted>/switzerlandnorth/8099ad13-8961-4a7a-9087-934e16f489d4
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Ratelimit-Remaining-Subscription-Global-Reads: 3749
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Ratelimit-Remaining-Subscription-Reads: 249
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Request-Id: 177f4f4e-db06-4169-acd8-67f7ecfb43bb
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Ms-Routing-Request-Id: SWITZERLANDNORTH:20260605T102251Z:177f4f4e-db06-4169-acd8-67f7ecfb43bb
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: X-Msedge-Ref: Ref A: 25A1D3CF7C4F42F1ADFDC13B0A13067E Ref B: GVA201070811042 Ref C: 2026-06-05T10:22:51Z
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: [DEBUG] Unlocking "/subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/commit|switzerlandnorth|Connectivity"
2026-06-05T12:22:51.338+0200 [DEBUG] provider.terraform-provider-azurerm_v4.76.0_x5: [DEBUG] Unlocked "/subscriptions/<Redacted>/resourceGroups/rg-resources/providers/Microsoft.Network/networkManagers/avnm-chn-hccase/commit|switzerlandnorth|Connectivity"
2026-06-05T12:22:51.338+0200 [TRACE] provider.terraform-provider-azurerm_v4.76.0_x5: Called downstream: tf_req_id=62a43e2f-57f1-7afc-8307-e49ce4a5388c @caller=github.com/hashicorp/terraform-plugin-sdk/v2@v2.38.1/helper/schema/resource.go:981 @module=sdk.helper_schema tf_mux_provider="*schema.GRPCProviderServer" tf_provider_addr=registry.terraform.io/hashicorp/azurerm tf_resource_type=azurerm_network_manager_deployment tf_rpc=ApplyResourceChange timestamp="2026-06-05T12:22:51.338+0200"
2026-06-05T12:22:51.338+0200 [TRACE] provider.terraform-provider-azurerm_v4.76.0_x5: Received downstream response: tf_proto_version=5.10 tf_provider_addr=registry.terraform.io/hashicorp/azurerm tf_req_id=62a43e2f-57f1-7afc-8307-e49ce4a5388c tf_resource_type=azurerm_network_manager_deployment @module=sdk.proto diagnostic_error_count=1 diagnostic_warning_count=0 tf_req_duration_ms=12274 tf_rpc=ApplyResourceChange @caller=github.com/hashicorp/terraform-plugin-go@v0.29.0/tfprotov5/internal/tf5serverlogging/downstream_request.go:61 timestamp="2026-06-05T12:22:51.338+0200"
2026-06-05T12:22:51.338+0200 [ERROR] provider.terraform-provider-azurerm_v4.76.0_x5: Response contains error diagnostic: tf_provider_addr=registry.terraform.io/hashicorp/azurerm tf_rpc=ApplyResourceChange @caller=github.com/hashicorp/terraform-plugin-go@v0.29.0/tfprotov5/internal/diag/diagnostics.go:58 @module=sdk.proto diagnostic_summary="creating Network Manager Deployment: (Scope Access \"Connectivity\" / Location \"switzerlandnorth\" / Network Manager \"avnm-chn-hccase\" / Resource Group \"rg-resources\"): polling after NetworkManagerCommitsPost: internal-error: polling support for the Content-Type \"\" was not implemented: <nil>" tf_proto_version=5.10 tf_req_id=62a43e2f-57f1-7afc-8307-e49ce4a5388c tf_resource_type=azurerm_network_manager_deployment diagnostic_detail="creating Network Manager Deployment: (Scope Access \"Connectivity\" / Location \"switzerlandnorth\" / Network Manager \"avnm-chn-hccase\" / Resource Group \"rg-resources\"): polling after NetworkManagerCommitsPost: internal-error: polling support for the Content-Type \"\" was not implemented: <nil>" diagnostic_severity=ERROR timestamp="2026-06-05T12:22:51.338+0200"
2026-06-05T12:22:51.338+0200 [TRACE] provider.terraform-provider-azurerm_v4.76.0_x5: Served request: tf_resource_type=azurerm_network_manager_deployment @caller=github.com/hashicorp/terraform-plugin-go@v0.29.0/tfprotov5/tf5server/server.go:957 @module=sdk.proto tf_provider_addr=registry.terraform.io/hashicorp/azurerm tf_rpc=ApplyResourceChange tf_proto_version=5.10 tf_req_id=62a43e2f-57f1-7afc-8307-e49ce4a5388c timestamp="2026-06-05T12:22:51.338+0200"
2026-06-05T12:22:51.339+0200 [TRACE] maybeTainted: azurerm_network_manager_deployment.connectivity encountered an error during creation, so it is now marked as tainted
2026-06-05T12:22:51.339+0200 [TRACE] terraform.contextPlugins: Schema for provider "registry.terraform.io/hashicorp/azurerm" is in the global cache
2026-06-05T12:22:51.339+0200 [TRACE] NodeAbstractResourceInstance.writeResourceInstanceState to workingState for azurerm_network_manager_deployment.connectivity
2026-06-05T12:22:51.339+0200 [TRACE] NodeAbstractResourceInstance.writeResourceInstanceState: removing state object for azurerm_network_manager_deployment.connectivity
2026-06-05T12:22:51.339+0200 [TRACE] evalApplyProvisioners: azurerm_network_manager_deployment.connectivity is tainted, so skipping provisioning
2026-06-05T12:22:51.339+0200 [TRACE] maybeTainted: azurerm_network_manager_deployment.connectivity was already tainted, so nothing to do
2026-06-05T12:22:51.339+0200 [TRACE] terraform.contextPlugins: Schema for provider "registry.terraform.io/hashicorp/azurerm" is in the global cache
2026-06-05T12:22:51.339+0200 [TRACE] NodeAbstractResourceInstance.writeResourceInstanceState to workingState for azurerm_network_manager_deployment.connectivity
2026-06-05T12:22:51.339+0200 [TRACE] NodeAbstractResourceInstance.writeResourceInstanceState: removing state object for azurerm_network_manager_deployment.connectivity
{"@level":"info","@message":"azurerm_network_manager_deployment.connectivity: Creation errored after 12s","@module":"terraform.ui","@timestamp":"2026-06-05T12:22:51.339275+02:00","hook":{"resource":{"addr":"azurerm_network_manager_deployment.connectivity","module":"","resource":"azurerm_network_manager_deployment.connectivity","implied_provider":"azurerm","resource_type":"azurerm_network_manager_deployment","resource_name":"connectivity","resource_key":null},"action":"create","elapsed_seconds":12},"type":"apply_errored"}
2026-06-05T12:22:51.339+0200 [DEBUG] State storage *cloud.State declined to persist a state snapshot
2026-06-05T12:22:51.339+0200 [ERROR] vertex "azurerm_network_manager_deployment.connectivity" error: creating Network Manager Deployment: (Scope Access "Connectivity" / Location "switzerlandnorth" / Network Manager "avnm-chn-hccase" / Resource Group "rg-resources"): polling after NetworkManagerCommitsPost: internal-error: polling support for the Content-Type "" was not implemented: <nil>
2026-06-05T12:22:51.339+0200 [TRACE] vertex "azurerm_network_manager_deployment.connectivity": visit complete, with errors
2026-06-05T12:22:51.339+0200 [TRACE] dag/walk: upstream of "provider[\"registry.terraform.io/hashicorp/azurerm\"] (close)" errored, so skipping
2026-06-05T12:22:51.339+0200 [TRACE] dag/walk: upstream of "root" errored, so skipping
2026-06-05T12:22:51.339+0200 [TRACE] terraform.contextPlugins: Schema for provider "registry.terraform.io/hashicorp/azurerm" is in the global cache
2026-06-05T12:22:51.339+0200 [TRACE] terraform.contextPlugins: Schema for provider "registry.terraform.io/hashicorp/vault" is in the global cache
2026-06-05T12:22:51.339+0200 [TRACE] terraform.contextPlugins: Schema for provider "registry.terraform.io/azure/azapi" is in the global cache
2026-06-05T12:22:51.339+0200 [TRACE] terraform.contextPlugins: Schema for provider "registry.terraform.io/infobloxopen/infoblox" is in the global cache
2026-06-05T12:22:51.340+0200 [DEBUG] cloud/state: state read serial is: 513; serial is: 513
2026-06-05T12:22:51.340+0200 [DEBUG] cloud/state: state read lineage is: 0109da17-9b4d-f84e-d781-eda7c4eb4d99; lineage is: 0109da17-9b4d-f84e-d781-eda7c4eb4d99
2026-06-05T12:22:51.529+0200 [TRACE] Intermediate state interval is set by header to 0s
2026-06-05T12:22:51.609+0200 [TRACE] Skipping intermediate state interval because Content-Type was ""
2026-06-05T12:22:51.638+0200 [TRACE] Skipping intermediate state interval because Content-Type was ""
2026-06-05T12:22:51.693+0200 [TRACE] Intermediate state interval is set by header to 0s
{"@level":"warn","@message":"Warning: Value for undeclared variable","@module":"terraform.ui","@timestamp":"2026-06-05T12:22:51.694433+02:00","diagnostic":{"severity":"warning","summary":"Value for undeclared variable","detail":"The root module does not declare a variable named \"vault_appservice_secret_id\" but a value was found in file \"/home/tfc-agent/.tfc-agent/component/terraform/runs/run-rggw3nnxw7UjSgTm/terraform.tfvars\". If you meant to use this value, add a \"variable\" block to the configuration.\n\nTo silence these warnings, use TF_VAR_... environment variables to provide certain \"global\" settings to all configurations in your organization. To reduce the verbosity of these warnings, use the -compact-warnings option."},"type":"diagnostic"}
{"@level":"warn","@message":"Warning: Value for undeclared variable","@module":"terraform.ui","@timestamp":"2026-06-05T12:22:51.694558+02:00","diagnostic":{"severity":"warning","summary":"Value for undeclared variable","detail":"The root module does not declare a variable named \"vault_appservice_role_id\" but a value was found in file \"/home/tfc-agent/.tfc-agent/component/terraform/runs/run-rggw3nnxw7UjSgTm/terraform.tfvars\". If you meant to use this value, add a \"variable\" block to the configuration.\n\nTo silence these warnings, use TF_VAR_... environment variables to provide certain \"global\" settings to all configurations in your organization. To reduce the verbosity of these warnings, use the -compact-warnings option."},"type":"diagnostic"}
{"@level":"error","@message":"Error: creating Network Manager Deployment: (Scope Access \"Connectivity\" / Location \"switzerlandnorth\" / Network Manager \"avnm-chn-hccase\" / Resource Group \"rg-resources\"): polling after NetworkManagerCommitsPost: internal-error: polling support for the Content-Type \"\" was not implemented: \u003cnil\u003e","@module":"terraform.ui","@timestamp":"2026-06-05T12:22:51.694932+02:00","diagnostic":{"severity":"error","summary":"creating Network Manager Deployment: (Scope Access \"Connectivity\" / Location \"switzerlandnorth\" / Network Manager \"avnm-chn-hccase\" / Resource Group \"rg-resources\"): polling after NetworkManagerCommitsPost: internal-error: polling support for the Content-Type \"\" was not implemented: \u003cnil\u003e","detail":"creating Network Manager Deployment: (Scope Access \"Connectivity\" / Location \"switzerlandnorth\" / Network Manager \"avnm-chn-hccase\" / Resource Group \"rg-resources\"): polling after NetworkManagerCommitsPost: internal-error: polling support for the Content-Type \"\" was not implemented: \u003cnil\u003e","address":"azurerm_network_manager_deployment.connectivity","range":{"filename":"gh_case.tf","start":{"line":36,"column":62,"byte":1305},"end":{"line":36,"column":63,"byte":1306}},"snippet":{"context":"resource \"azurerm_network_manager_deployment\" \"connectivity\"","code":"resource \"azurerm_network_manager_deployment\" \"connectivity\" {","start_line":36,"highlight_start_offset":61,"highlight_end_offset":62,"values":[]}},"type":"diagnostic"}
2026-06-05T12:22:51.695+0200 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2026-06-05T12:22:51.704+0200 [INFO]  provider: plugin process exited: plugin=.terraform/providers/registry.terraform.io/hashicorp/azurerm/4.76.0/linux_amd64/terraform-provider-azurerm_v4.76.0_x5 id=362
2026-06-05T12:22:51.704+0200 [DEBUG] provider: plugin exited
Operation failed: failed running terraform apply (exit 1)
```

### Expected Behaviour

terraform apply completes successful

### Actual Behaviour

terraform apply fails during the polling phase of the Network Manager Commit operation with:

 Error: waiting for Network Manager Commit / deployment: error: polling support for the Content-Type "" was not implemented

### Steps to Reproduce

 1. terraform init
 2. terraform apply

### Important Factoids

running in Switzerlandnorth

### References

The issue started appearing for us between June 1 and 2, 2026. Pinning the provider to 4.73.0 resolves it. The changelogs for the newer versions don't mention any changes to azurerm_network_manager_deployment directly, but 4.74.0 (released May 21, 2026) included a go-azure-sdk bump to v0.20260520.1145341 (#32420) maybe this is causing this in combination with arm api changes ?

References
 - Release 4.74.0 changelog: https://github.com/hashicorp/terraform-provider-azurerm/blob/main/CHANGELOG.md
 - go-azure-sdk bump PR: https://github.com/hashicorp/terraform-provider-azurerm/pull/32420

--- a/internal/services/network/custompollers/network_manager_deployment_poller.go
+++ b/internal/services/network/custompollers/network_manager_deployment_poller.go
@@ -1,0 +1,62 @@
+// Copyright IBM Corp. 2014, 2025
+// SPDX-License-Identifier: MPL-2.0
+
+package custompollers
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/network/2025-01-01/networkmanagers"
+	"github.com/hashicorp/go-azure-sdk/sdk/client/pollers"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/network/parse"
+)
+
+var _ pollers.PollerType = &networkManagerDeploymentPoller{}
+
+type networkManagerDeploymentPoller struct {
+	client *networkmanagers.NetworkManagersClient
+	id     parse.ManagerDeploymentId
+}
+
+func NewNetworkManagerDeploymentPoller(client *networkmanagers.NetworkManagersClient, id parse.ManagerDeploymentId) pollers.PollerType {
+	return &networkManagerDeploymentPoller{
+		client: client,
+		id:     id,
+	}
+}
+
+func (p networkManagerDeploymentPoller) Poll(ctx context.Context) (*pollers.PollResult, error) {
+	result := &pollers.PollResult{
+		PollInterval: 10 * time.Second,
+		Status:       pollers.PollingStatusInProgress,
+	}
+
+	payload := networkmanagers.NetworkManagerDeploymentStatusParameter{
+		Regions:         &[]string{location.Normalize(p.id.Location)},
+		DeploymentTypes: &[]networkmanagers.ConfigurationType{networkmanagers.ConfigurationType(p.id.ScopeAccess)},
+	}
+
+	networkManagerId := networkmanagers.NewNetworkManagerID(p.id.SubscriptionId, p.id.ResourceGroup, p.id.NetworkManagerName)
+	resp, err := p.client.NetworkManagerDeploymentStatusList(ctx, networkManagerId, payload)
+	if err != nil {
+		result.Status = pollers.PollingStatusFailed
+		return result, pollers.PollingFailedError{
+			Message: fmt.Errorf("listing deployments for %s", networkManagerId).Error(),
+		}
+	}
+
+	// List request may initially return no results
+	if resp.Model == nil || resp.Model.Value == nil || len(*resp.Model.Value) == 0 {
+		return result, nil
+	}
+
+	if *(*resp.Model.Value)[0].DeploymentStatus == networkmanagers.DeploymentStatusDeployed {
+		result.Status = pollers.PollingStatusSucceeded
+		return result, nil
+	}
+
+	return result, nil
+}

--- a/internal/services/network/custompollers/network_manager_deployment_poller.go
+++ b/internal/services/network/custompollers/network_manager_deployment_poller.go
@@ -8,6 +8,7 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
 	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
 	"github.com/hashicorp/go-azure-sdk/resource-manager/network/2025-01-01/networkmanagers"
 	"github.com/hashicorp/go-azure-sdk/sdk/client/pollers"
@@ -53,7 +54,7 @@ func (p networkManagerDeploymentPoller) Poll(ctx context.Context) (*pollers.Poll
 		return result, nil
 	}
 
-	if *(*resp.Model.Value)[0].DeploymentStatus == networkmanagers.DeploymentStatusDeployed {
+	if pointer.From((*resp.Model.Value)[0].DeploymentStatus) == networkmanagers.DeploymentStatusDeployed {
 		result.Status = pollers.PollingStatusSucceeded
 		return result, nil
 	}

--- a/internal/services/network/network_manager_deployment_resource.go
+++ b/internal/services/network/network_manager_deployment_resource.go
@@ -13,9 +13,11 @@ import (
 	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
 	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
 	"github.com/hashicorp/go-azure-sdk/resource-manager/network/2025-01-01/networkmanagers"
+	"github.com/hashicorp/go-azure-sdk/sdk/client/pollers"
 	"github.com/hashicorp/terraform-provider-azurerm/helpers/azure"
 	"github.com/hashicorp/terraform-provider-azurerm/internal/locks"
 	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/network/custompollers"
 	"github.com/hashicorp/terraform-provider-azurerm/internal/services/network/parse"
 	"github.com/hashicorp/terraform-provider-azurerm/internal/services/network/validate"
 	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
@@ -138,8 +140,17 @@ func (r ManagerDeploymentResource) Create() sdk.ResourceFunc {
 				CommitType:       networkmanagers.ConfigurationType(state.ScopeAccess),
 			}
 
-			// TODO: implement callback, requires migrating to an ID implementing `resourceids.ResourceId`
-			if err := client.NetworkManagerCommitsPostThenPoll(ctx, *networkManagerId, input); err != nil {
+			// The API has an issue where it may return an async operation URL that is invalid, breaking the default LRO poller
+			if _, err := client.NetworkManagerCommitsPost(ctx, *networkManagerId, input); err != nil {
+				return fmt.Errorf("creating %s: %+v", id, err)
+			}
+
+			if metadata.Client.Features.PersistIDOnCreateBeforePollingForCompletion {
+				metadata.SetID(id)
+			}
+
+			poller := pollers.NewPoller(custompollers.NewNetworkManagerDeploymentPoller(client, *id), time.Second*10, pollers.DefaultNumberOfDroppedConnectionsToAllow)
+			if err := poller.PollUntilDone(ctx); err != nil {
 				return fmt.Errorf("creating %s: %+v", id, err)
 			}
 

